routing/index_road_graph.cpp

```diff
@@ -78,8 +85,14 @@ void IndexRoadGraph::GetEdges(Junction const & junction, bool isOutgoing, TEdgeV
     if (IndexGraphStarter::IsFakeSegment(segment))
       continue;
 
-    edges.emplace_back(FeatureID(m_mwmId, segment.GetFeatureId()), segment.IsForward(),
-                       segment.GetSegmentIdx(), GetJunction(segment, false /* front */),
+    platform::CountryFile const & file = m_numMwmIds->GetFile(segment.GetMwmId());
+    MwmSet::MwmHandle const handle = m_index.GetMwmHandleByCountryFile(file);
+    if (!handle.IsAlive())
+      MYTHROW(RoutingException, ("Can't get mwm handle for", file));
+
+    edges.emplace_back(FeatureID(MwmSet::MwmId(handle.GetInfo()), segment.GetFeatureId()),
```

Comment:
So, the code is called during routing reconstruction. On this stage you should know all mwms the route passes through. Is it possible to get MwmIds in advance, for all needed mwms?